docs/kikoff-guide.mdx

```diff
@@ -697,6 +696,215 @@ One particularly valuable piece of metadata is **`verificationMethod`**, which d
+                    - `birthDate`
+                    - `address.line1`
+                    - `address.line2`
+                    - `address.city`
+                    - `address.state`
+                    - `address.zipCode`
+                    - `address.country`
+                    - `ssn`
+                </details>
+        - **Data** in this context means identity information about a user (e.g. their name, contact information, etc.)
+        - **Metadata** in this context means any additional data that's _not_ identity information about a user (e.g. the verification method for their identity information).
+            - `metadataType` is the type of metadata. 
+                <details>
+                <summary>Accepted Values</summary>
+
+                Currently, possible values for `metadataType` are:
+                    - `id`: unique identifier for a credential
+                        - UUID
+                        - Applies only to credentials
+                    - `verificationMethod`: how an identifier or credential was verified
+                        - enum: `"phone_carrier"`, `"otp"`, `"self_attested"`, or `null` 
+                        - Applies to both identifiers and credentials
+                    - `status`: status of a credential
+                        - enum: `"valid"` or `"revoked"`
+                        - Applies only to credentials
+                    - `issuerUuid`: brand that issued a credential
+                        - UUID
+                        - Applies only to credentials
+                    - `issuanceDate`: date a credential was issued
+                        - Unix timestamp
+                        - Applies only to credentials
+                    - `expirationDate`: date a credential expires
+                        - Unix timestamp
+                        - Applies only to credentials
+                </details>
+    </details>
+
+    ```typescript title="Data the User Shared"
+    {
+        "identifiers": {
+            "phone": "+101234567890"
+        },
+        "credentials": {
+            "email": "richard@piedpiper.net",
+            "fullName": {
+                "firstName": "Richard",
+                "lastName": "Hendricks"
+            },
+            "birthDate": "08/01/1989",
+            "address": {
+                "line1": "5320 Newell Rd",
+                "city": "Palo Alto",
+                "state": "CA",
+                "zip": "94303",
+                "country": "US"
+            },
+            "ssn": "123456789"
+        },
+        "metadata": {
+            "identifiers": {
+                "verificationMethod": {
+                    "phone": "phone_carrier"
+                }
+            },
+            "credentials": {
+                "id": {
+                    "email": "8a1d4e35-413d-496b-b499-8810b55cfb5c",
+                    "fullName": {
+                        "firstName": "2e6a7b9a-e93e-43ba-98a9-c554f4e16457",
```

Comment:
Hmm I really wonder why an Acceptor would care about the a credential `id` 🤔. I suppose in the spirt of returning everything we already were this is the correct way to go.

Reply:
Yeah...